test/image_test.py

```diff
@@ -806,13 +806,55 @@ def test_load_basic(self):
         
         f.close()
 
-    def todo_test_load_extended(self):
+    def test_load_extended(self):
 
         # __doc__ (as of 2008-08-02) for pygame.image.load_extended:
 
         # pygame module for image transfer
 
-        self.fail()
+        #We test loading the following file types: bmp, png, jpg, gif (non-animated), tga (uncompressed), tif, xpm, ppm, pgm
+        #All the loaded images are smaller than 32 x 32 pixels
+
+        #test loading a bmp
```

Comment:
We only recently added support for using subTest inside the pygame test suite, but they're excellent! With them you can make the messages better if something fails. Also it allows the further parts to be tested if something at the top of the test fails (for example if the png loading fails all the subTests after that will run). Have a look at the docs for subTest: https://docs.python.org/3/library/unittest.html#subtests

Something similar to this... (untested):
```python
with self.subTest("test loading a bmp", filename=filename, expected_color=(...)):
    surf = pygame.image.load_extended(example_path(filename))
    self.assertEqual(surf.get_at((0,0)), expected_color)
```

It's also possible to loop over a data structure of [[filename, expected_color], ...]